docs/guide/robot_sbc/setup_jetson_nano.md

```diff
@@ -66,76 +136,385 @@ pip3 install -U gdown
-pip3 install torch-1.8.0-cp36-cp36m-linux_aarch64.whl
-sudo apt-get install libjpeg-dev zlib1g-dev libpython3-dev libavcodec-dev libavformat-dev libswscale-dev
-mkdir -p ~/projects; cd ~/projects
-git clone -b v0.9.0 https://github.com/pytorch/vision torchvision
-cd torchvision 
-python setup.py install
-cd ../
+mkdir projects
+cd ~/projects
+git clone https://github.com/autorope/donkeycar
+cd donkeycar
+git fetch --all --tags -f
+latestTag=$(git describe --tags `git rev-list --tags --max-count=1`)
+git checkout $latestTag
+pip install -e .[nano45]
+
 ```
 
-Optionally, you can install the RPi.GPIO clone for Jetson Nano from [here](https://github.com/NVIDIA/jetson-gpio). This is not required for default setup, but can be useful if using LED or other GPIO driven devices.
+### Step 5a: (Optional) Install PyGame for USB camera
+
+If you plan to use a USB camera, you will also want to setup pygame:
+
+```bash
+sudo apt-get install python-dev libsdl1.2-dev libsdl-image1.2-dev libsdl-mixer1.2-dev libsdl-ttf2.0-dev libsdl1.2-dev libsmpeg-dev python-numpy subversion libportmidi-dev ffmpeg libswscale-dev libavformat-dev libavcodec-dev libfreetype6-dev
+pip install pygame
+```
 
+Later on you can add the `CAMERA_TYPE="WEBCAM"` in myconfig.py.
 
 
-##  Step 5: Install Donkeycar Python Code
 
-* Change to a dir you would like to use as the head of your projects. Assuming you've already made the `projects` directory above, you can use that:
+## Installation for Donkey Car main
+
+Instructions for the latest code from the `main` branch. Note the 
+installation differs between the two available OSs. On Jetson Nano you will 
+need to install Jetpack 4.6.2. If you have newer Jetson board or a Jetson 
+Xavier you need to install Jetpack 5.0.2.
+
+
+### Installation on Jetson Nano
+
+
+* [Step 1b: Flash Operating System](#step-1b-flash-operating-system)
+* [Step 2b: Free up the serial port](#step-2b-free-up-the-serial-port-optional-only-needed-if-youre-using-the-robohat-mm1)
+* [Step 3b: Setup Python Environment](#step-3b-setup-python-environment)
+* [Step 4b: (Optional) Install PyGame for USB camera](#step-4b-optional-install-pygame-for-usb-camera)
+
+
+#### Step 1b: Flash Operating System
+
+These instructions work for Jetpack 4.6.2 on 4GB Nano and 2GB Nano using 
+different disk images.
+
+* If you have a 4gb Jetson Nano then download Jetpack 4.6.2 from Nvidia
+  here: [jetson-nano-jp461-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jeston_nano/jetson-nano-jp461-sd-card-image.zip)
+
+* If you have a 2gb Jetson Nano the download Jetpack 4.6.2 from Nvidia
+  here: [jetson-nano-2gb-jp461-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jetson_nano_2gb/jetson-nano-2gb-jp461-sd-card-image.zip)
+
+If you are on a previous version like 4.6.1 you can upgrade with:
+
+```commandline
+sudo apt update
```

Comment:
I just sat 20 minutes waiting for this to finish... maybe we should add the `-y` flag: `sudo apt update -y` `sudo apt upgrade -y`.

I haven't tested it yet, particularly the fullscreen dialog: "Configuring docker.io" "Automatically restart Docker daemon?"

Reply:
`-y` flag did _not_ work to automatically accept prompts.



~75%
```
Setting up nvidia-l4t-core (32.7.3-20221122092935) ...

Configuration file '/etc/ld.so.conf.d/nvidia-tegra.conf'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** nvidia-tegra.conf (Y/I/N/O/D/Z) [default=N] ? 
```

77% 

```
Setting up nvidia-l4t-oem-config (32.7.3-20221122092935) ...

Configuration file '/etc/systemd/nv-oem-config-post.sh'
 ==> Deleted (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** nv-oem-config-post.sh (Y/I/N/O/D/Z) [default=N] ? 
```


81%. This is a fullscreen prompt
```
 Configuring docker.io 
Automatically restart Docker daemon?      
```